testsuite/MathMLToDisplay/General/Colors/issue1002.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html class="reftest-wait">
+  <head>
+    <title>Test SVG output inherting color</title>
+    <!-- Copyright (c) 2015 MathJax Consortium
+         License: Apache License 2.0 -->
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+    <script type="text/javascript" src="../../../header.js"></script>
+    <script type="text/javascript" src="../../../scriptTests.js"></script>
+    <script type="text/javascript">
+      function getTestCases() {
+      var path = document.querySelectorAll('g')[0];
+      var check1 = (path.getAttribute('fill') === 'currentColor' & path.getAttribute('stroke') === 'currentColor');
+      var computedColor = window.getComputedStyle(path).color;
```

Comment:
My bad! I missed the original comment. Yes, I was aware that these don't work on IE<9 and chose to disregard this (mostly because I once ran into trouble with `getElementsByTagName` for SVG elements and never had the time to figure it out properly on IE8). 

Is it ok as is?